bank-sdk/example-app/src/androidTest/java/net/gini/android/bank/sdk/exampleapp/ui/screens/DigitalInvoiceEditButton.kt

```diff
@@ -0,0 +1,167 @@
+package net.gini.android.bank.sdk.exampleapp.ui.screens
+
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.clearText
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.action.ViewActions.replaceText
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import androidx.test.uiautomator.UiCollection
+import androidx.test.uiautomator.UiSelector
+import org.hamcrest.Matchers.allOf
+
+class DigitalInvoiceEditButton {
+
+    fun checkEditButtonTitleIsDisplayed() :Boolean {
+        val uiCollection =
+            UiCollection(UiSelector().className("android.view.ViewGroup"))
+        val editButton = uiCollection.getChildByInstance(
+            UiSelector().className("android.widget.TextView")
+                .text("Edit")
+                .resourceId("net.gini.android.bank.sdk.exampleapp:id/gbs_edit_button"), 0)
```

Comment:
Can we change this to R.id.gbs_edit_button?

Reply:
Done.

Reply:
In `resourceId` we cannot pass **`R.id.gbs_edit_button`** since it accepts String parameter in the function..